scripts/lit.js

```diff
@@ -6,20 +6,39 @@
 // Configuration:
 //
 // Commands:
-// randbot lit <message_link> - Immortalizes a message in #lit... fam
+//   hubot lit <message_link> - Immortalizes a message in #lit... fam
+//   hubot lit - Immortalizes a shared message in #lit... fam
 //
 // Author:
 //   mikestephens/harasho
 
+const { failureMessages, successMessages } = require('./utils/lit.messages');
+
+// TODO: make this a configurable values
+// paro: Just for the time being, let's keep these separate.
+const channel = 'lit';
+const validURL = 'https://hashtaggaming.slack.com/archives/';
+
+function hasAttachments(rawMessage) {
```

Comment:
Pull in this library called Lodash and you can simply do 

```js
const _ = require('lodash');

const hasAttachments = rawMessage => _.get(rawMessage, 'attachments.length', 0) > 0;
```

Reply:
ugh... I don't know... Do we want to pull in and be dependent on lodash just for that one job?

Reply:
Yeah, definitely. Lodash is one of those libraries that I pull in my default. I'm surprised it's not already in this project. The total built is about 24kB and it's not something that adds to the start time for the project.